<?xml version="1.0" ?>
<launch>
	<include file="$(find hullify)/launch/rpi.machine" />

	<!-- Start up image processing on the main computer side -->
	<node name="republish" type="republish" pkg="image_transport" args="compressed in:=camera/image raw out:=camera/image_decompressed" />
	<node name="image_view" type="image_view" pkg="image_view" args="image:=camera/image_decompressed"/>

	<!-- Start up the raspberry pi's camera -->
	<node machine="pi" name="raspicam_node" type="raspicam_node" pkg="raspicam" output="screen"/>

	<!-- Start up the takktiles -->
	<node machine="pi" name="takktile_node" type="takktile_node.py" pkg="takktile_ros" output="screen"/>
</launch>
